model/config.go

```diff
@@ -578,7 +578,9 @@ func (o *Config) SetDefaults() {
 
 	if o.ServiceSettings.WebserverMode == nil {
 		o.ServiceSettings.WebserverMode = new(string)
-		*o.ServiceSettings.WebserverMode = "regular"
+		*o.ServiceSettings.WebserverMode = "gzip"
+	} else if *o.ServiceSettings.WebserverMode == "regular" {
+		*o.ServiceSettings.WebserverMode = "gzip"
```

Comment:
There is a new option `uncompressed` you can see that here: https://github.com/mattermost/platform/pull/3458/files#diff-469d87b06f69ef98e3dfa8a1f785570bR77

This line is so any current Mattermost server will default to using gzip.